CMakeLists.txt

```diff
@@ -59,6 +59,11 @@ include(make/policies.cmake NO_POLICY_SCOPE)
 # this must be done prior to the project() command and the var
 # must be set in the cache.
 if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio")
+  set(CMAKE_SYSTEM_VERSION_TEMP  $ENV{WindowsSDKVersion})
```

Comment:
suggest: A comment explaining that this is a workaround until CMake issue 17992 is fixed would be helpful.